packages/core/rollup.config.js

```diff
@@ -50,6 +50,8 @@ const createConfig = ({ input, output, external, env, min = false, useESModules
     }),
     babel({
       exclude: 'node_modules/**',
+      // Es modules in browser does not need transpilation
```

Comment:
This is not quite true, I'd prefer to transpile this - just not to the es5. We could leverage https://babeljs.io/docs/en/babel-preset-env#targetsesmodules .

This would make the setup slightly more complicated (how to turn it on & off in babelrc files?), but maybe we could leverage upcoming `.custom` feature of `rollup-plugin-babel` (available in v5.0.0-alpha.0), I don't have time right now to investigate this, but it would look smth like this:
```
babel.custom(() => ({
  config(config) {
    // manipulate the config here (it should contain `@babel/preset-env`)
    // or just return untouched config back for every other build
    return config
  }
}))
```